tests/utils/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2024 Renesas Electronics Corporation.
+# Copyright (C) 2024 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET utils_test)
+
+# ######################################################################################################################
+# Sources
+# ######################################################################################################################
+
+set(SOURCES exception_test.cpp)
+
+# ######################################################################################################################
+# Includes
+# ######################################################################################################################
+
+# ######################################################################################################################
+# Target
+# ######################################################################################################################
+
+add_library(${TARGET} STATIC ${SOURCES})
+
+# ######################################################################################################################
+# Labraries
+# ######################################################################################################################
+
+target_link_libraries(${TARGET} PUBLIC aoscommon GTest::gtest)
```

Comment:
in theory it should be enough to link utils static library & aoscommon will be linked transitively

Reply:
this is on theory. Practically, aos_common is not linked.